drivers/virtio/net/virtio_net.c

```diff
@@ -486,7 +494,7 @@ static int virtio_netdev_rxq_enqueue(struct uk_netdev_rx_queue *rxq,
 	/**
 	 * Retrieve the buffer header length.
 	 */
-	rc = uk_netbuf_header(netbuf, header_sz);
+	rc = uk_netbuf_header(netbuf, sizeof(*rxhdr));
```

Comment:
I think the space for prepending should be either 10 or 12, depending on the virtio version.

Reply:
This actually needs to use the padded header. According to the commit history:
```
    We introduce buffer padding while transmitting and receiving data. We
    explicitly introduce padding to make sure that each network packet
    needs two descriptors (one for virtio header and another for
    network packet).

    According the specification 5.1.6.6, we need to explicitly use 2
    descriptor for each transmit and receive network packet since we do not
    negotiate for VIRTIO_F_ANY_LAYOUT.
    
    In the previous implementation the sg list merged the two descriptor
    into one as the memory region were contiguous.
```